docs/milestone-1-validation.md

```diff
@@ -0,0 +1,99 @@
+# Milestone-1 Validation
+
+* Scality Object Storage Interoperability Service (for short, **Scality OSIS**)
+* VMware Cloud Director Object Storage Extension (for short, **vCloud Director OSE**)
+
+## Prerequisites
+
+1. Run Vault (check [How to Run Vault](https://github.com/scality/Vault/blob/development/7.10/TESTING.md#how-to-run-1))
+   
+1. Run Cloudserver (check [running Cloudserver here](https://github.com/scality/cloudserver/tree/development/7.10#installation)) with env variable: `S3VAULT=scality` so that it can use vault as auth backend.
+
+1. Update environment file `application.properties` with Vault target URL (before starting Scality OSIS application)
+    * Vault URL should be set at property `osis.scality.vault.endpoint` (Refer Sample `application.properties` [here](../src/main/resources/application.properties))
+    * Vault Super admin credentials should be updated at properties `osis.scality.vault.access-key` and `osis.scality.vault.secret-key`
+
+1. Run the Scality OSIS application (Refer [here](../README.md#running-the-application)). Milestone-1 release tag `0.1.0-SNAPSHOT`
+
+1. On OSE machine, vCloud Director OSE and OSE UI should be installed and running successfully.
+
+1. On OSE machine, integrate Scality OSIS with vCloud Director OSE. (Refer [here](#OSIS-integration-with-vCloud-Director-OSE))
+
+## Validation
+
+1. Login as a vCloud Director Administrator on vCloud Director UI
+1. Navigate to `Resources` tab.
+1. Create a new `Organization` (a.k.a vCloud Director tenant) with required fields.
+1. Navigate to `Object Storage` tab.
+1. `Tenants` pane should be activated and should list all the vCloud Director tenants with associated `Storage Tenant ID` (if exists)
+1. Click on the Tenant name that is created in step 2 under `Tenants` pane.
+1. Enable toggle button beside the Tenant name.
+1. Click `Enable`. (This step will create a Vault Account in the background)
+1. A `Storage Tenant ID` should be generated on the UI which is also the associated Vault Account's Account ID.
+
+### OSIS integration with vCloud Director OSE:
+To integrate vCloud Director OSE and the Scality OSIS application:
+1. Configure a connection between vCloud Director OSE and the **admin** service of the Scality OSIS instance.
+   * Provide OSIS URL with port number when prompted for admin URL.
+   * Enter Vault Super admin credentials when prompted for credentials.
+```shell
+[root@vcdose centos]# ose osis admin set
+OSIS Compliant Platform Name : Scality
+URL      : https://localhost:8443
+Username : D4IT2AWSB588GO5J9T00
+Secret   : ****************************************
```

Comment:
Did you intentionally show a bad secret for the example or what is the point of this?

Reply:
Those are the Vault Super admin credentials and if our Vault super admin secret key does not meet the complexity criteria we should still continue as we must provide Vault super admin credentials here.

Reply:
I feel like the term "criteria" is wrong then, maybe "recommendation". If something said I do not meet the criteria I would assume that meant I couldn't do something and had to try something else, not just hit the ok button anyway.

Reply:
Correct but that is the error from Vnware OSE, which is not our product. These are only steps to integrate OSE with out product OSIS

Reply:
Whelp, can't object to that then lol